File: 2 Revalida en C++/5 Funciones/6.cpp
///funcion de la serie de fibonacci
#include <bits/stdc++.h>
using namespace std;
int fibonacci(int n)
{
    int x;
    int y;
    int valor;
    if((n==0)||(n==1))
        {
		    return n;
		}
	else
        {
            x=0;
            y=1;
            for (int i=2;i<=n;i++)
                {
                    y=y+x;
                    x=y-x;
                }
            valor=y;
        }
    return valor;
}
int main()
{
    int n;
    cout<<"ingrese n:";
    cin>>n;
    cout<<"el numero de la serie de fiboacci es = "<<fibonacci(n)<<endl;
    return 0;
}

File: 1 Revalida en python/5 Funciones/6.py
#funcion de la serie de fibonacci
def fibonacci(n):
	if((n==0)or(n==1)):
		return n
	else:
		x=0
		y=1
		i=1
		for i in range(i,n):
			y=y+x
			x=y-x
		valor=y
		return valor
n=int(input("ingrese un numero:"))
print fibonacci(n)